frontend/src/lib/components/Table/private-components/tableRow.tsx

```diff
@@ -25,7 +25,7 @@ export function TableRow<T extends Record<string, any>>(props: TableRowProps<T>)
 
     return (
         <tr
-            className={resolveClassNames("group/tr border-b-2 last:border-b-0", {
+            className={resolveClassNames("group/tr border-b-2 last:border-b-0 select-none", {
```

Comment:
In my usage I find it handy to be able to copy some of the row data, but this disables possibility to mark and copy text. I assume this is the solution to prevent marking of text when using `shift`+`click` when performing multiselect of rows?
If so, is there another way to allow marking for copy, but prevent the text marking when multiselecting?